docs/installation-getting-started/linux.mdx

```diff
@@ -96,7 +96,7 @@ To run Pieces for Developers on Linux, be sure to download both of the applicati
 
 3. Install the Pieces Desktop Snap:
     ```shell
-    $ sudo snap install pieces-for-developers
+    sudo snap install pieces-for-developers
```

Comment:
@shivay-at-pieces there are some other locations where this may still be present - do you think having them/not having them matters? @chottuthejimmy has brought this up in the past as well.

Reply:
Hi @jordan-pieces I couldn't find any other such $ being used with a command in the docs repo. 
But I do agree with @chottuthejimmy if any developer has not configured their terminal to accept the $ being copied with the command it will continously throw an error and they will have to copy the command manually.